mappings/net/minecraft/class_7428.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/class_7428
```

Comment:
Name this `net.minecraft.network.encryption.PublicKey` or something

Reply:
bad idea, `PublicKey` is a class from java that is referenced in there. I named it `PlayerPublicKey`